rust/core-lib/src/plugins/rpc.rs

```diff
@@ -179,14 +187,44 @@ pub enum PluginRequest {
 #[serde(tag = "method", content = "params")]
 /// RPC commands sent from plugins.
 pub enum PluginNotification {
-    AddScopes { scopes: Vec<Vec<String>> },
-    UpdateSpans { start: usize, len: usize, spans: Vec<ScopeSpan>, rev: u64 },
-    Edit { edit: PluginEdit },
-    Alert { msg: String },
-    AddStatusItem { key: String, value: String, alignment: String },
-    UpdateStatusItem { key: String, value: String },
-    RemoveStatusItem { key: String },
-    ShowHover { request_id: usize, result: Result<Hover, RemoteError> },
+    AddScopes {
```

Comment:
what's up with this? Is this like a new rustfmt change? I'm confused

Reply:
Yeah, that was just a rustfmt change. Nothing to do with annotations